qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/FindPatientSteps.java

```diff
@@ -64,7 +64,7 @@ public void returnResults() {
 		assertNotNull(firstPatientIdentifier);
 	}
 
-	@And("User clicks on first patient")
+	@And("User clicks on returned patient")
```

Comment:
Any reason for this change?

Reply:
As @dkayiwa  said above, we need to say `click On returned patient `when a user searches using the patient identifier irrespectively.